backend/data_tools/environment/dev.py

```diff
@@ -0,0 +1,15 @@
+from data_tools.environment.common import DataToolsConfig
+
+
+class DevConfig(DataToolsConfig):
+    @property
+    def db_connection_string(self):
+        return "postgresql://postgres:local_password@localhost:5432/postgres"  # pragma: allowlist secret
+
+    @property
+    def opre_excel_connection_string(self):
+        return "excel:///?Excel File='/Users/jdeangelis/PycharmProjects/OPRE-OPS-2/data-tools/data/REDACTED - FY22 Budget Summary -10-12-22.xlsm'"
```

Comment:
Save here - default to a user directory?

Reply:
Checked in accidentally - backed it out.